### ISSUE TITLE
Turn off lights on sleep/lock not working in Linux/Ubuntu

### ISSUE THREAD

**Issue by fgblomqvist** [psieg/Lightpack#150]
If you set the 'Scene luminosity threshhold' to 0 the lights will turn off when I lock, simply because the monitor goes off. If I don't set it to 0, the lights will stay on but at the minimum brightness I've set.

Sleeping the computer leaves the lights at the same color/brightness they are on my lockscreen (when the monitor is on, say, when I am typing in my password etc.). I saw #98 so I thought that maybe a linux-implementation is missing as well? I'm guessing there's a DBus event to listen for, but one way to do it until such an implementation in Prismatik is in place is to place this script in `/etc/pm/sleep.d/`:

```
#!/bin/bash
case $1 in
suspend)
    #suspending to RAM
    prismatik --off
    ;;
resume)
    #resume from suspend
    sleep 3
    prismatik --on
    ;;
esac
```

**Comment by psieg** [comment]
Should be fixed since #225

**Comment by psieg** [comment]
Actually #225 fixed only MacOS, sorry
